ruby-app/.env.example

```
@@ -1 +1,6 @@
-OPENWEATHER_API_KEY=
\ No newline at end of file
+OPENWEATHER_API_KEY=
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+DATABASE_URL=
+MONITORING_IP=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)